src/server/server.js

```diff
@@ -1,22 +1,34 @@
 import {SocketCluster} from 'socketcluster';
-import {auth0Secret} from './secrets';
 import path from 'path';
+import dotenv from 'dotenv';
+import dotenvExpand from 'dotenv-expand';
 // import os from 'os';
 
+// Import .env and expand variables:
+try {
+  const myEnv = dotenv.config({silent: true});
+  dotenvExpand(myEnv);
+} catch (e) {
+  console.warn(`Unable to load .env: ${e}`);
+}
+
 // const numCpus = os.cpus().length;
 export const options = {
-  authKey: new Buffer(auth0Secret, 'base64'),
+  authKey: process.env.AUTH0_CLIENT_SECRET ?
+    new Buffer(process.env.AUTH0_CLIENT_SECRET, 'base64') :
+    new Buffer('BksPeQQrRkXhDrugzQDg5Nw-IInub9RkQ-pSWohUM9s6Oii4xoGVCrK2_OcUCfYZ', 'base64'),
   logLevel: 1,
   // change this to scale vertically
   workers: 1,
   brokers: 1,
-  port: 3000,
+  host: process.env.HOST || '127.0.0.1',
```

Comment:
It's always provided by Dokku; but could check above and pop a warning if it's not. That'd be cleaner.